ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/v2/UserManagementStore.scala

```diff
@@ -125,9 +125,10 @@ object UserManagementStore {
 
   case class UserInfo(user: User, rights: Set[UserRight])
 
-  sealed trait Error extends RuntimeException
+  sealed trait Error
```

Comment:
Why is there an asymmetry between `UserManagementStore.Error` and `PartyRecordStore.Error`?

Reply:
Forget to remove 'extends RuntimeException' from the other one.
Having 'extends RuntimeException' is useless here and moreover it prevents us from getting a nice case class' toString method